src/borrow_tracker/mod.rs

```diff
@@ -0,0 +1,337 @@
+pub mod stacked;
+
+use std::cell::RefCell;
+use std::fmt;
+use std::num::NonZeroU64;
+
+use log::trace;
+use smallvec::SmallVec;
+
+use rustc_data_structures::fx::{FxHashMap, FxHashSet};
+use stacked::diagnostics::RetagCause;
+
+use crate::*;
```

Comment:
```suggestion

use stacked::diagnostics::RetagCause;
use crate::*;
```
The organization is
- std imports
- other dependency imports
- rustc imports
- local crate imports

with an empty line between each block.